tchannel-example/src/main/java/com/uber/tchannel/thrift/SetValueHandlerDefault.java

```diff
@@ -38,17 +35,13 @@ public SetValueHandlerDefault(Map<String, String> keyValueStore) {
         this.keyValueStore = keyValueStore;
     }
 
-    @Override
-    public Response<KeyValue.setValue_result> handle(Request<KeyValue.setValue_args> request) {
+    public KeyValue.setValue_result handleImpl(KeyValue.setValue_args setValueArgs) {
```

Comment:
Yes, I have had some thoughts about it and I was planning to wrap it with Request and Response but then I found it strange that the handlers should know about transport headers or ttl. On top of that they have their own application headers. But yeah lets braintstorm on that.